recipe/meta.yaml

```diff
@@ -19,6 +21,7 @@ requirements:
     # perl is required to run the tests on UNIX.
     - perl  # [unix]
     - pkg-config  # [unix]
+    - patch
```

Comment:
If a patch will be applied on all platforms then we need also the `win`-variant `m2-patch`, so in `req/build` should be:
```
- patch   # [not win]
- m2-patch  # [win]
```

Reply:
Added the lines mentioned above!